password_security/models/res_company.py

```diff
@@ -14,11 +14,6 @@ class ResCompany(models.Model):
         default=60,
         help="How many days until passwords expire",
     )
-    password_length = fields.Integer(
-        "Characters",
-        default=12,
-        help="Minimum number of characters",
-    )
```

Comment:
Do you need to provide a migration for this field? Maybe something like:

```
ICP = self.env['ir.config_parameter']
ICP.set_param('auth_password_policy.minlength', max(
    ICP.get_param('auth_password_policy.minlength'),
    *self.env['res.company'].search([]).mapped('password_length'))
))
```

Reply:
Yes, seems a good idea :+1:  Thanks!